pkg/neotest/basic.go

```diff
@@ -401,6 +412,10 @@ func TestInvoke(bc *core.Blockchain, tx *transaction.Transaction) (*vm.VM, error
 	ttx := *tx
 	ic, _ := bc.GetTestVM(trigger.Application, &ttx, b)
 
+	if isCoverageEnabled() {
+		ic.VM.SetOnExecHook(coverageHook())
+	}
+
```

Comment:
That's what was meant by alternative in https://github.com/nspcc-dev/neo-go/pull/3462#issuecomment-2236429892, we can document the way it is now and it'd be sufficient. If it ever to become insufficient this can be done. `Blockchain` has too many things already, so I'd try to avoid adding more methods there.